pages.zh/common/gpg-zip.md

```diff
@@ -3,7 +3,7 @@
 > 使用`GPG`加密存档中的文件和目录。
 > 更多信息： <https://www.gnupg.org/documentation/manuals/gnupg/gpg_002dzip.html>.
 
-- 使用密码将一个目录加密为`archive.gpg`：
+- 使用密码将一个目录加密为`archive.gpg`:
 
 `gpg-zip --symmetric --output {{archive.gpg}} {{path/to/directory}}`
```

Comment:
```suggestion
`gpg-zip --symmetric --output {{档案.gpg}} {{路径/目录}}`
```

Reply:
I found the main purpose of this PR is to fix punctuations, so I'll ignore translation problems from now on.

Reply:
Yeah, I think this PR is already big enough 😅 
But I feel like translating untranslated arguments in Chinese pages would be worth it's own PR